jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -61,6 +61,13 @@ def load_profiles(self, secret_name="jupyter-singleuser-profiles", filename=None
       profiles_config_maps.extend(sorted(self.openshift.get_config_maps_matching_label()))
       for cm_name in profiles_config_maps:
         config_map_yaml = self.openshift.read_config_map(cm_name, key_name)
+        if type(config_map_yaml) != dict:
+          message = '''
+          Ignoring CM %s - content must be object, not list.
```

Comment:
We are not ignoring the CM in the end, so the error message should be something like:

```
ConfigMap %s is incorrect - content must be object, not list
```